src/tracer.js

```diff
@@ -263,6 +281,7 @@ export default class Tracer {
      *        was successful.
      */
     flush(done) {
```

Comment:
can we remove this from public API?  It was decided against in other languages, and this method does not always make sense, e.g. in case of in-memory only span collectors.